app/screens/post_options/post_options.js

```diff
@@ -15,7 +15,9 @@ import {t} from 'app/utils/i18n';
 import {showModal, dismissModal} from 'app/actions/navigation';
 
 import {isSystemMessage} from 'mattermost-redux/utils/post_utils';
-import {OPTION_HEIGHT, getInitialPosition} from './post_options_utils';
+import {OPTION_HEIGHT, REACTION_PICKER_HEIGHT, getInitialPosition} from './post_options_utils';
+import ReactionPicker from '../../components/reaction_picker';
```

Comment:
```suggestion
import ReactionPicker from 'app/components/reaction_picker';
```

and should be above the import for `./post_options_utils`